SU-SWS/decanter
# ADAPT-1763 Adjust card border and add border stroke colors

# READY FOR REVIEW

# Summary
- Adjust card border default color and add border stroke colors

# Needed By (Date)
- 08/06

# Urgency
- low

# Steps to Test

1. Pull this branch
2. Create test files in `/developer` (see readme.Md)
3. Pull in card template, json, and scss. Adjust scss to use default box-shadow variables.
4. Run local dev and inspect

# Associated Issues and/or People
- [ADAPT-1763](https://stanfordits.atlassian.net/browse/ADAPT-1763)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
